pkg/networkservice/common/updatetoken/server_test.go

```diff
@@ -224,7 +224,7 @@ func (f *updateTokenServerSuite) TestChain() {
 
 	got, err := server.Request(context.Background(), request)
 	require.Equal(t, 3, len(got.Path.PathSegments))
-	require.Equal(t, 2, int(got.Path.Index))
+	require.Equal(t, 0, int(got.Path.Index))
```

Comment:
Context:
https://github.com/networkservicemesh/sdk/blob/217f0e054ba9b68a0c7ae02420eb86169da0f714/pkg/networkservice/common/updatetoken/server_test.go#L214-L227

`Path.Index` is now restored by updatetoken on line 215.
This is changed because of the restored nesting order.